apps/fluent-tester/src/FluentTester/TestComponents/ContextualMenu/ContextualMenuTest.tsx

```diff
@@ -38,13 +38,12 @@ const contextualMenu: React.FunctionComponent<{}> = () => {
     (key) => {
       setLastMenuItemClicked(key);
     },
-    [setLastMenuItemClicked]
+    [setLastMenuItemClicked],
```

Comment:
for my information, is this syntax we are encouraging even for last entry?

Reply:
Yes. It was added by our linter extension. There was some discussion about it before we made it one of the rules. I touched this file at some point, so the linter added all of these.